cmake/motor.cmake

```diff
@@ -0,0 +1,13 @@
+set(MOTOR_SRC_PATH ${CMAKE_SOURCE_DIR}/src)
+
+set(MOTOR_SOURCES
+    ${MOTOR_SRC_PATH}/Motor.c
+)
+
+add_library(motor_stl STATIC
+    ${MOTOR_SOURCES}
+)
+
+target_include_directories(motor_stl PUBLIC 
+    ${CMAKE_SOURCE_DIR}/inc
```

Comment:
Has it become common to use 'inc' instead of 'include' as directory name?  If not, I prefer full words (as in the 'target_include_directories' directive.

My rule of thumb -- If I NEVER abbreviate whole words, I NEVER have to guess what abbreviation I use.  Example 'fn'.  I'd use full_name, first_name, file_name, or function_name.  I have a little brain, and can use as many hints as I can get.